Helper/Data.php

```diff
@@ -716,8 +735,13 @@ public function getLiveEndpointPrefix($storeId = null)
      */
     public function getCheckoutFrontendRegion($storeId = null)
     {
-        $prefix = trim($this->getAdyenAbstractConfigData('checkout_frontend_region', $storeId));
-        return $prefix;
+        $checkoutFrontendRegion = $this->getAdyenAbstractConfigData('checkout_frontend_region', $storeId);
+
+        if (is_null($checkoutFrontendRegion)) {
+            return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
            return null;
```